src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -396,48 +348,43 @@ void AbstractWriteLog<I>::load_existing_entries(DeferredContexts &later) {
     } else if (pmem_entry->is_write()) {
       ldout(m_image_ctx.cct, 20) << "Entry " << entry_index
                                  << " is a write. pmem_entry=[" << *pmem_entry << "]" << dendl;
-      auto write_entry =
+      auto write_entry =    
```

Comment:
Nit: why all the whitespace added at the end of these lines (here and below)?

Reply:
sorry, that was a mistake. Cleaned up now.